packages/frontend/src/components/wallet-migration/modals/MigrateAccountsModal/SelectDestinationWallet.jsx

```diff
@@ -79,6 +80,13 @@ export const WALLET_OPTIONS = shuffle([
         getUrl: ({ hash }) => `finer://wallet.near.org/batch-import#${hash}`,
         checkAvailability: () => isMobile('iOS'),
     },
+    {
+        id: 'welldone-wallet',
+        name: 'WELLDONE Wallet',
+        icon: <img src={ImgWelldoneWallet} alt="WELLDONE Wallet Logo" />,
+        getUrl: ({ hash }) => `https://docs.welldonestudio.io/contribution/202211-batch-import#${hash}`,
+        checkAvailability: () => true,
```

Comment:
one last request, is it possible to add `ledgerSupport: false,` after line 88?
This is new boolean we introduced, to indicate ledger compatibility

Reply:
ok, I added it! please check it.